src/Core/Appplication/Resources/config/services.yaml

```diff
@@ -123,8 +123,10 @@ services:
     tags: ["action.after_command_executed"]
   project_workflow:
     public: true
+    shared: false
```

Comment:
what does it mean this setting?

Reply:
Each time container is asked for this service it will give a new instance. This behavior enables us to init multiple (root and nested) workflows at the same time.